lib/foreman_tasks/tasks/cleanup.rake

```diff
@@ -3,21 +3,21 @@ namespace :foreman_tasks do
     desc <<DESC
 Clean tasks based on filter and age. ENV variables:
 
-  * FILTER     : scoped search filter (example: 'label = "Actions::Foreman::Host::ImportFacts"')
-  * AFTER      : delete tasks created after *AFTER* period. Expected format is a number followed by the time unit (s,h,m,y), such as '10d' for 10 days
-  * STATES     : comma separated list of task states to touch with the cleanup, by default only stopped tasks are covered
-  * NOOP       : set to "true" if the task should not actuall perform the deletion
-  * VERBOSE    : set to "true" for more verbose output
-  * BATCH_SIZE : the size of batches the tasks get processed in (1000 by default)
+  * TASK_SEARCH : scoped search filter (example: 'label = "Actions::Foreman::Host::ImportFacts"')
+  * AFTER       : delete tasks created after *AFTER* period. Expected format is a number followed by the time unit (s,h,m,y), such as '10d' for 10 days
+  * STATES      : comma separated list of task states to touch with the cleanup, by default only stopped tasks are covered
+  * NOOP        : set to "true" if the task should not actuall perform the deletion
+  * VERBOSE     : set to "true" for more verbose output
+  * BATCH_SIZE  : the size of batches the tasks get processed in (1000 by default)
 
-If none of FILTER, BEFORE, STATES is specified, the tasks will be cleaned based
+If none of TASK_SEARCH, BEFORE, STATES is specified, the tasks will be cleaned based
 configuration in settings
 DESC
     task :run => 'environment' do
       options = {}
 
-      if ENV['FILTER']
```

Comment:
Yes, to preven't a situation, where somebody would have some filter set to
delete
some tasks, and after this change, we would just delete everything because
no filter was set

-- Ivan

On Tue, Jun 28, 2016 at 3:11 PM, David Davis <notifications@github.com>
wrote:

> In lib/foreman_tasks/tasks/cleanup.rake
> <https://github.com/theforeman/foreman-tasks/pull/186#discussion_r68752680>
> :
>
> >  configuration in settings
> >  DESC
> >      task :run => 'environment' do
> >        options = {}
> >
> > -      if ENV['FILTER']
>
> You mean like a deprecation error (e.g. Please use TASK_SEARCH instead of
> FILTER)?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/theforeman/foreman-tasks/pull/186/files/8ea8416c5ab033f5acbe939621d0affe8f17702b#r68752680>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AALn61kL7oMxZUVeoI2mW7XyWsRlAxyNks5qQR2OgaJpZM4Ifo-9>
> .
>